__tests__/app.test.js

```diff
@@ -404,4 +404,19 @@ describe("All endpoints", () => {
 				})
 		})
 	})
+
+	//-----#12 POST /api/comments/:comment_id endpoint ----------
+	describe.only("DELETE - /api/comments/:comment_id", () => {
+		test("Status 204 responds with empty body ", () => {
+			return request(app).delete("/api/comments/1").expect(204)
```

Comment:
we could go a step further here as its in test to check to see if the comment has actually been removed